### PR TITLE
fix(#188): Made child picture not mandatory when adding a new child

### PR DESCRIPTION
**Pull request by ckelwin**
Fixes JordyHers-org/Times-up-flutter#188:

Made child picture not mandatory when adding a new child. A default icon will be displayed as a substitute.

image
image
image

### CODE REVIEW

File: lib/app/features/parent_side/child_details_page.dart [hunk 1]
```diff
@@ -128,53 +128,52 @@ class _ChildDetailsPageState extends State<ChildDetailsPage>
           return [
             SliverAppBar(
               actions: [
-                if (model.image != null)
```
Comment (ckelwin) [inline lib/app/features/parent_side/child_details_page.dart:131]: Changes on this file mainly removing if statement on line 131, and reformatting the indentation of the code.

File: lib/app/features/parent_side/child_details_page.dart [hunk 2]
```diff
@@ -128,53 +128,52 @@ class _ChildDetailsPageState extends State<ChildDetailsPage>
...
+                            ],
                           ),
                         ),
-                        child: ClipOval(
-                          child: Image.network(model.image!),
-                        ).p4,
                       ),
-                    ],
-                  )
-                else
```
Comment (ckelwin) [inline lib/app/features/parent_side/child_details_page.dart:176]: (changes) .... and removing this else part.

Then updated the ClipOval image below to either use image uploaded or a hardcoded icon.

File: lib/app/features/parent_side/edit_child_page.dart [hunk 3]
```diff
@@ -89,32 +89,33 @@ class _EditChildPageState extends State<EditChildPage> {
...
-      if (localFile == null) return;
       setState(() {
         appState = AppState.loading;
       });
 
       id = uuid.v4().substring(0, 8).toUpperCase();
-      try {
-        final fileExtension = path.extension(localFile.path);
-        JHLogger.$.d(fileExtension);
+      if(localFile!=null) {
```
Comment (ckelwin) [inline lib/app/features/parent_side/edit_child_page.dart:97]: Changes on this file, (if my memory is right):
- removing if statement on line 92.
- Added this if null check that deals with the image uploaded.

**Review by JordyHers — APPROVED** [review]
LGTM

**Pull request merged**